src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -137,7 +141,18 @@ export const TimeSeries = ({
         animateData={false}
         onPointerUpdate={handleCursorUpdate}
         theme={[
-          chartTheme,
+          {
+            crosshair: {
+              ...chartTheme.crosshair,
+            },
+            axes: {
+              tickLabel: {
+                padding: {
+                  inner: hasVisibleAnnotations ? 19 : chartTheme.axes.tickLabel.padding.inner,
```

Comment:
@alexwizp what this 19 indicate? The padding ? Can you add it in a constant first to be more readable?

Reply:
done